docs/components/image.html

```diff
@@ -0,0 +1,140 @@
+                </a>
+            </div>
+            <div class="nav-right">
+                <ul>
+                    <li><a href="../../index.html" class="nav-right-link">Home <i class="fas fa-home"></i></a></li>
+                    <li><a href="../introduction.html" class="nav-right-link">Docs <i class="fas fa-file-word"></i></a>
+                    </li>
+                    <li><a href="https://github.com/tanishq20" target="_blank" class="nav-right-link">GitHub <i
+                                class="fab fa-github"></i></a></li>
+                </ul>
+            </div>
+        </nav>
+
+        <div class="doc-container">
+            <aside class="sidebar">
+                <div class="sidebar-heading">Getting Started</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="#">Introduction</a></li>
+                    <li class="sidebar-items-link"><a href="#">Installation</a></li>
+                </ul>
+                <div class="sidebar-heading">Components</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="./avatar.html">Avatar</a></li>
+                    <li class="sidebar-items-link"><a href="./badge.html">Badge</a></li>
+                    <li class="sidebar-items-link"><a href="./alert.html">Alert</a></li>
+                    <li class="sidebar-items-link"><a href="./button.html">Button</a></li>
+                    <li class="sidebar-items-link"><a href="#">Card</a></li>
+                    <li class="sidebar-items-link active-state"><a href="./image.html">Image</a></li>
+                    <li class="sidebar-items-link"><a href="#">Input</a></li>
+                </ul>
+                <div class="sidebar-heading">Customization</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="#">Theme</a></li>
+                    <li class="sidebar-items-link"><a href="#">Palette</a></li>
+                    <li class="sidebar-items-link"><a href="#">Typography</a></li>
+                </ul>
+                <div class="sidebar-heading">Overview</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="#">Conclusion</a></li>
+                    <li class="sidebar-items-link"><a href="#">Get involved</a></li>
+                </ul>
+            </aside>
+
+            <main class="main-content">
+                <h1 class="main-head">Image</h1>
+                <p class="main-desc">Images that work well on devices with widely differing screen sizes and
+                    resolutions.
+                </p>
+                <div class="section-change"></div>
+
+                <!-- Responsive Image -->
+
+                <h3 class="main-subhead">Responsive Image</h3>
+                <p class="main-desc">To obtain this result, use <i>img</i> tag inside a <i>div</i> tag with

```

Comment:
use code instead of `i` tag

```
<code>....</code>
```

Reply:
Oh didn't know this `code` tag. 
Will change it directly from the dev branch so it will reflect on all the pages.